### PR TITLE
Ifaa 20 8

### PR DESCRIPTION
**Pull request by asiatqc**
Taras, review please.

### CODE REVIEW

File: src/test/java/com/softserveinc/edu/ita/enums/ordering_page/SearchMethods.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.softserveinc.edu.ita.enums.ordering_page;
+
+/**
+ * Created by i07016 on 06.07.2015.
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/edu/ita/enums/ordering_page/SearchMethods.java]: palka - remove

File: src/test/java/com/softserveinc/edu/ita/enums/ordering_page/SortFields.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.softserveinc.edu.ita.enums.ordering_page;
+
+/**
+ * Created by i07016 on 06.07.2015.
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/edu/ita/enums/ordering_page/SortFields.java]: palka - remove

File: src/test/java/com/softserveinc/edu/ita/pageobjects/AddItemPage.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+        for (WebElement itemRow : itemsRows) {
+            rowsCells = itemRow.findElements(AddItemPageLocators.ITEMS_ROW_CELL.getBy());
+            String[] item = {rowsCells.get(0).getText(), rowsCells.get(1).getText()};
+            itemsList.add(item);
+        }
+        return itemsList;
+    }
+
+    public AddItemPage selectSearchMethod(SearchMethods searchMethod) {
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/edu/ita/pageobjects/AddItemPage.java]: selectSearchCondition()

File: src/test/java/com/softserveinc/edu/ita/pageobjects/AddItemPage.java [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+                break;
+            case SEARCH_BY_DESCRIPTION:
+                searchMethodSelect.selectByIndex(1);
+                break;
+        }
+
+        return this;
+    }
+
+    public boolean isItemsListSorted(List<String[]> itemsList, List<Product> productList) {
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/edu/ita/pageobjects/AddItemPage.java]: not an action, move out of page object

File: src/test/java/com/softserveinc/edu/ita/pageobjects/NewOrderPage.java [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+import java.math.BigDecimal;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+/**
+ * PageObject Class presents page for new Order creating
+ */
+public class NewOrderPage extends LogOutBase {
+
+    //private Random randomGenerator = new Random();
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/edu/ita/pageobjects/NewOrderPage.java]: palka

File: src/test/java/com/softserveinc/edu/ita/pageobjects/NewOrderPage.java [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+        sendKeys(NewOrderPageLocators.CREDIT_CARD_NUMBER_INPUT, creditCardNumber);
+        return this;
+    }
+
+    public NewOrderPage fillCVV2NumberField(String CVV2) {
+        sendKeys(NewOrderPageLocators.CVV2_INPUT, CVV2);
+        return this;
+    }
+
+    // fills Preferable Delivery Date field with date later of current for 0-30 days
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/edu/ita/pageobjects/NewOrderPage.java]: turn into java doc

File: src/test/java/com/softserveinc/edu/ita/pageobjects/NewOrderPage.java [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+
+    public boolean isLastButtonEnabled() {
+        return isElementEnabled(NewOrderPageLocators.NAVIGATION_LAST_BUTTON);
+    }
+
+    public void clickForwardButton() {
+        click(NewOrderPageLocators.NAVIGATION_FORWARD_BUTTON);
+    }
+
+    public void selectShowLinesCount(int option) {
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/edu/ita/pageobjects/NewOrderPage.java]: make an enum out of option

File: src/test/java/com/softserveinc/edu/ita/enums/ordering_page/SearchMethods.java (hunk 1 above)
Comment (asiatqc) [inline src/test/java/com/softserveinc/edu/ita/enums/ordering_page/SearchMethods.java]: removed

File: src/test/java/com/softserveinc/edu/ita/pageobjects/AddItemPage.java (hunk 3 above)
Comment (asiatqc) [inline src/test/java/com/softserveinc/edu/ita/pageobjects/AddItemPage.java]: renamed also with locator, enum and local variables

File: src/test/java/com/softserveinc/edu/ita/pageobjects/AddItemPage.java (hunk 4 above)
Comment (asiatqc) [inline src/test/java/com/softserveinc/edu/ita/pageobjects/AddItemPage.java]: moved out to the test, made it private

File: src/test/java/com/softserveinc/edu/ita/pageobjects/NewOrderPage.java (hunk 5 above)
Comment (asiatqc) [inline src/test/java/com/softserveinc/edu/ita/pageobjects/NewOrderPage.java]: yeah

File: src/test/java/com/softserveinc/edu/ita/pageobjects/NewOrderPage.java (hunk 6 above)
Comment (asiatqc) [inline src/test/java/com/softserveinc/edu/ita/pageobjects/NewOrderPage.java]: turned

File: src/test/java/com/softserveinc/edu/ita/enums/ordering_page/SortFields.java [hunk 8]
```diff
@@ -0,0 +1,9 @@
+package com.softserveinc.edu.ita.enums.ordering_page;
+
+/**
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/edu/ita/enums/ordering_page/SortFields.java]: remove all useless comments

**Pull request merged**